packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -164,20 +155,9 @@ export default {
             commit, state, dispatch, rootGetters
         }, { url, data, timeout }) => {
             // TODO: deal with exceptions and handle this action properly (when the functionality is ready)
-            const authHeader = state.authToken && `Bearer ${state.authToken}`;
-            const experimentationHeaders = rootGetters[`${VUEX_CHECKOUT_EXPERIMENTATION_MODULE}/getExperimentsHeaders`];
-            const config = {
-                headers: {
-                    'Content-Type': 'application/json',
-                    ...(state.authToken && {
-                        Authorization: authHeader
-                    }),
-                    ...experimentationHeaders
-                },
-                timeout
-            };
 
-            const { data: responseData, headers } = await axios.patch(url, data, config);
+            const { data: responseData, headers } = await checkoutApi.updateCheckout(url, state, rootGetters, data, timeout);
```

Comment:
SHould we catch HTTP exception here?

Reply:
I haven't added a catch as it wasn't in the original axios request - The catches are actually happening in the `handleErrorState` in `Checkout.vue` so I _think_ that's why we don't need them here 👍